src/modules/commander/commander_params.c

```diff
@@ -629,6 +629,16 @@ PARAM_DEFINE_INT32(COM_ARM_MAG_ANG, 45);
  */
 PARAM_DEFINE_INT32(COM_ARM_MAG_STR, 1);
 
+/**
+ * Enable arming grace.
```

Comment:
```suggestion
 * Rearming grace period
```

Slightly more self-explanatory? You can skip "Enable" because it should be obvious with the corresponding boolean metadata.